package/CHANGELOG

```diff
@@ -36,6 +36,8 @@ Changes
   * GSD is no longer a core dependency of MDAnalysis. It must be installed
```

Comment:
I forgot, can you also add your handle in the list of authors for 2.6.0 on line 16 please?